src/wymeditor/editor/native-edit-registration.js

```diff
@@ -51,7 +51,9 @@ WYMeditor.NativeEditRegistration.prototype._onAnyNativeEdit = function () {
         undoRedo = nativeEditRegistration.wym.undoRedo;
 
     // remove redo points
-    undoRedo.history.changesetsFore = [];
+    if (undoRedo.history) {
+        undoRedo.history.forgetAllForward();
```

Comment:
The [new API–foo](https://github.com/PolicyStat/object-history#forgetallforward).